experimental/traffic-portal/src/app/core/cache-groups/asns/detail/asn-detail.component.html

```diff
@@ -29,6 +29,11 @@
 				<mat-select name="cachegroup" [(ngModel)]="asn.cachegroupId" required>
 					<mat-option *ngFor="let cachegroup of cachegroups" [value]="cachegroup.id">{{cachegroup.name}}</mat-option>
 				</mat-select>
+				<mat-hint>
+					<a mat-icon-button [disabled]="!asn.cachegroupId" class="small-icon-button" matTooltip="View Cache Group Details" aria-label="View Cache Group Details" color="primary" [routerLink]="'/core/cache-groups/' + asn.cachegroupId" target="_blank">
```

Comment:
You don't need to change this, but `aria-label` is unnecessary here, because `matTooltip` will add an `aria-describedby` pointing to an element with the given text.